cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -946,6 +946,11 @@ ReconciliationState createReconciliationState(Reconciliation reconciliation, Kaf
             ));
         }
 
+        /* test */
+        public void setZkAncillaryCmChange(boolean zkAncillaryCmChange) {
```

Comment:
if you are adding this just for test, you should not need the `public` modifier but `package` level should be ok. This is a sort of pattern we are using in other places where a method is added just for test.
So you can remove `public` and move the `/* test */` comment to its place.